### ISSUE TITLE
Header icon broken since 0.61.0

### ISSUE THREAD

**Issue by panki27** [Textualize/textual#4534]
As already reported in Discord: since upgrading to `textual==0.61.0`, clicking the header icon does no longer open the command palette.
The palette can still be summoned by hitting the keyboard shortcut.

MRE: 
```py
from textual import __version__
from textual.app import App, ComposeResult
from textual.widgets import Label, Header

class CommandPaletteApp(App[None]):

    def compose(self) -> ComposeResult:
        yield Header()
        yield Label(f"This is Textual v{__version__}")

if __name__ == "__main__":
    CommandPaletteApp().run()
```

Clicking the header icon with the mouse results in the following message being logged:
```
<action> 'command_palette' has no target. Could not find methods '_action_command_palette' or 'action_command_palette'
```

Darren and Dave were able to narrow it down do this line of code:
https://github.com/Textualize/textual/blob/964d988ac99fff1cad744b592844e4a5646023f5/src/textual/widgets/_header.py#L37

That should now be:
```
await self.run_action("app.command_palette")
```

**Comment by TomJGooding** [comment]
TIL that clicking the header icon will open the command pallette.

**Comment by willmcgugan** [comment]
Fixed in 0.63.0
